tests/components/oculus-touch-controls.test.js

```diff
@@ -10,38 +10,51 @@ suite('oculus-touch-controls', function () {
     el.setAttribute('oculus-touch-controls', '');
     el.addEventListener('loaded', function () {
       component = el.components['oculus-touch-controls'];
-      component.controllersWhenPresent = [
-        {id: 'Oculus Touch', index: 0, hand: 'left', pose: {}}
-      ];
+      // Initially no controllers are present
+      component.controllers = [];
+      // Our Mock data for enabling the controllers.
+      component.controllersWhenPresent = [{
+        id: 'Oculus Touch',
+        index: 0,
+        hand: 'left',
+        pose: {}
+      }];
       done();
     });
   });
 
   suite('checkIfControllerPresent', function () {
-    test('removes event listeners if controllers not present', function () {
```

Comment:
This test seems to be gone after the changes. Intentional?

Reply:
This test was invalid. The state it was testing for couldn't happen in production code. The correct state is that you add then remove a controller and then remove event listeners is called, which is covered by test 'removes event listeners if controller disappears' on line 90.